certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+                  "description": "startDate"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/api/v1/Organizations": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Add Organization",
+        "description": "Add Organization",
+        "operationId": "Add-Organization",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "AppToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "69a489d9-53fd-43aa-b04c-5c15ed4c73dd",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "UserToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "98ae7807-0902-4e66-9b0a-bfe0ecd204de",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "id",
+                  "title": "ID",
+                  "x-ms-visibility": "advanced"
+                },
+                "name": {
+                  "type": "string",
+                  "description": "name",
+                  "title": "Name",
+                  "x-ms-visibility": "advanced"
+                },
+                "showInDirectory": {
+                  "type": "boolean",
+                  "description": "show In Directory",
+                  "title": "Show In Directory?",
+                  "x-ms-visibility": "internal",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": false
+                },
+                "branchName": {
+                  "type": "string",
+                  "description": "branchName",
+                  "title": "Branch Name",
+                  "x-ms-visibility": "advanced"
+                },
+                "website": {
+                  "type": "string",
+                  "description": "website",
+                  "title": "Website",
+                  "x-ms-visibility": "advanced"
+                }
+              },
+              "required": [
+                "name",
+                "showInDirectory"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/api/v1/Individuals/{id}/Relationships": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@RjImpx Please 'replace' the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.